README.md

```diff
@@ -26,7 +26,7 @@ This site is created with Gatsby.
 
 1.  **Start developing.**
 
-    Mininal node version: `14.15.0` (for Gatsby v4)
+    Mininal node version: `16.13.0` (for Gatsby v4)
```

Comment:
Gasby v4 requires 14.5.0+ according to https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v3-to-v4/#minimal-nodejs-version-14150
The page doesn't mention any minima v16 version

Reply:
The minimal version we specify here is not only for the whole project ?
Because, we choose to build it only from v16.

Reply:
```suggestion
    Mininal node version: `16.x`
```